projects/schematics/src/shared/schematics-config-mappings.ts

```diff
@@ -0,0 +1,301 @@
+  TRACKING_AEP_SCHEMATICS_CONFIG,
+  TRACKING_GTM_SCHEMATICS_CONFIG,
+  TRACKING_PERSONALIZATION_SCHEMATICS_CONFIG,
+} from './lib-configs/tracking-schematics-config';
+import {
+  USER_ACCOUNT_SCHEMATICS_CONFIG,
+  USER_PROFILE_SCHEMATICS_CONFIG,
+} from './lib-configs/user-schematics-config';
+import { FeatureConfig, Module } from './utils/lib-utils';
+
+/**
+ * A list of all schematics feature configurations.
+ * _Must_ be updated when adding a new schematics
+ * library or a feature.
+ */
+export const SCHEMATICS_CONFIGS: FeatureConfig[] = [
+  // feature libraries start
+  ASM_SCHEMATICS_CONFIG,
+
+  CART_BASE_SCHEMATICS_CONFIG,
+  CART_IMPORT_EXPORT_SCHEMATICS_CONFIG,
+  CART_QUICK_ORDER_SCHEMATICS_CONFIG,
+  CART_WISHLIST_SCHEMATICS_CONFIG,
+  CART_SAVED_CART_SCHEMATICS_CONFIG,
+
+  CHECKOUT_BASE_SCHEMATICS_CONFIG,
+  CHECKOUT_B2B_SCHEMATICS_CONFIG,
+  CHECKOUT_SCHEDULED_REPLENISHMENT_SCHEMATICS_CONFIG,
+
+  ORDER_SCHEMATICS_CONFIG,
+
+  ORGANIZATION_ADMINISTRATION_SCHEMATICS_CONFIG,
+  ORGANIZATION_ORDER_APPROVAL_SCHEMATICS_CONFIG,
+
+  PRODUCT_CONFIGURATOR_TEXTFIELD_SCHEMATICS_CONFIG,
+  PRODUCT_CONFIGURATOR_RULEBASED_SCHEMATICS_CONFIG,
+  PRODUCT_CONFIGURATOR_CPQ_SCHEMATICS_CONFIG,
+
+  PRODUCT_BULK_PRICING_SCHEMATICS_CONFIG,
+  PRODUCT_IMAGE_ZOOM_SCHEMATICS_CONFIG,
+  PRODUCT_VARIANTS_SCHEMATICS_CONFIG,
+
+  QUALTRICS_SCHEMATICS_CONFIG,
+
+  SMARTEDIT_SCHEMATICS_CONFIG,
+
+  STOREFINDER_SCHEMATICS_CONFIG,
+
+  TRACKING_PERSONALIZATION_SCHEMATICS_CONFIG,
+  TRACKING_GTM_SCHEMATICS_CONFIG,
+  TRACKING_AEP_SCHEMATICS_CONFIG,
+
+  USER_ACCOUNT_SCHEMATICS_CONFIG,
+  USER_PROFILE_SCHEMATICS_CONFIG,
+
+  // integration libraries start
+  CDC_SCHEMATICS_CONFIG,
+
+  CDS_SCHEMATICS_CONFIG,
+
+  DIGITAL_PAYMENTS_SCHEMATICS_CONFIG,
+
+  EPD_SCHEMATICS_CONFIG,
+];
+
+/**
+ * Maps sub-features to their parent feature.
+ */
+export const {
+  /**
+   * Mapping of features to Spartacus library.
+   *
+   * E.g.:
+   *
+   * {
+   * ...,
+   *  '@spartacus/checkout': ['Checkout', 'Checkout-B2B', 'Checkout-Scheduled-Replenishment'],
+   * ...
+   * }
+   */
+  libraryFeatureMapping,
+  /**
+   * Mapping of feature-modules to the Spartacus library.
+   *
+   * E.g.:
+   *
+   * {
+   * ...,
+   * 'Checkout': ['CheckoutModule'],
+   * 'Checkout-B2B': ['CheckoutB2BModule'],
+   * 'Checkout-Scheduled-Replenishment': ['CheckoutScheduledReplenishmentModule'],
+   * ...
+   * }
+   */
+  featureFeatureModuleMapping,
+  /**
+   * Mapping of root feature-modules to the Spartacus library.
+   *
+   * E.g.:
+   *
+   * {
+   * ...,
+   * 'Checkout': ['CheckoutRootModule'],
+   * 'Checkout-B2B': ['CheckoutB2BRootModule'],
+   * 'Checkout-Scheduled-Replenishment': ['CheckoutScheduledReplenishmentRootModule'],
+   * ...
+   * }
+   */
+  featureRootModuleMapping,
+  /**
+   * Mapping of schematics configurations to the Spartacus library.
+   *
+   * E.g.:
+   *
+   * {
+   * ...,
+   * '@spartacus/checkout': [baseConfig, b2bConfig, scheduledReplenishmentConfig],
```

Comment:
think this doc is incorrect now compared to what it really did before

The mapping is feature-name to the entire config object instead of the mainscope
Checkout-B2b: entire-config-object from the CHECKOUT_B2B_SCHEMATICS_CONFIG that contains, lib, foldername,modulename, featuremodule, etc

Reply:
Yep, great catch, it's a leftover from previous version.